x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/action_type_menu.tsx

```diff
@@ -117,7 +123,14 @@ export const ActionTypeMenu = ({
       );
     });
 
-  return (
+  return loadingActionTypes ? (
```

Comment:
There was no loading indicator for the `AddConnector` flyout so on slow connections, it would show a blank flyout before loading all the action types. This adds a loading indicator.